src/graphql/resolvers/post.js

```diff
@@ -0,0 +1,153 @@
+/* eslint-disable */
+
+import helpers from '../helpers/post';
+import { EntityNotFoundError, ForbiddenError } from '../../errors';
+
+export default {
+  Query: {
+    async latest(parent, { params: args = {} }, { models: { post }, user }, info) {
+      args.latest = args.latest ? new Date(args.latest) : null;
+
+      const feedParams = helpers.getFeedParams({ post, user }, args, 'popularity');
+
+      if (!user) {
+        feedParams.filters = Object.assign({}, feedParams.filters, {
+          publications: { include: helpers.splitArrayStr(args.pubs) },
+          tags: { include: helpers.splitArrayStr(args.tags) },
+        });
+      }
+
+      return await post.generateFeed(feedParams);
+    },
+
+    async post(parent, args, { models, user }) {
```

Comment:
Please follow the same convention you used of object destructuring: `{ models: { post }, user }`